src/stories/Library/event-list-item/event-list-item.scss

```diff
@@ -0,0 +1,176 @@
+.event-list-item {
+  @include show-svg-on-hover();
+  text-decoration: none;
+  display: grid;
+  padding: $s-lg;
+  background-color: #fcfcfc;
```

Comment:
Please consider if this value is set right.

As far as I can tell in [Figma the background color of event-list-items`](https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?type=design&node-id=7477-45287&mode=design&t=Tyjdibqi0YgvStKK-0) is white.

Also: In general we should colors from `variables.colors.scss`. If the color is not already available we should add it.

Reply:
Apparently there never was a color for setting black white and grey as background-color: in the system. 

Background-colors using variables, were being set using the text variable. I've added black, white and grey global colors and refactored this. 

And yeah this should have be set to white from the start 🥳